server/src/routes/users.ts

```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import userController from "../controllers/userController";
+
+const router = express.Router();
+
+router
+  .get("/:id?", userController.getUsers)
```

Comment:
Add route for GET / to get one user and remove extra logic on userController.getUsers